bogota-abril-2019/cyclomatic-complexity/agePredictor.js

```diff
@@ -0,0 +1,14 @@
+
+
+//la complejidad del algoritmo es 2, se puede encontrar adjunta en el archivo cyclomatic complexity.jpg
+function predictAge(age1, age2, age3, age4, age5, age6, age7, age8) {
+    var ages = [...arguments];
```

Comment:
Nice aproach the spread over arguments.
But if do so, unnecesary use of explicit use of argument in the function declaration

Reply:
ok, if I understand, the correct use of arguments should be like this (¿?):

function predictAge(...arguments){